packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -109,6 +109,16 @@ export default class MainProcess {
       }
     );
 
+    const sharedProcessLogger = createLoggerService({
+      dev: this.settings.dev,
+      dir: this.settings.userDataDir,
+      name: 'shared',
+      passThroughMode: true,
+    });
+
+    sharedProcessLogger.pipeProcessOutputIntoLogger(this.sharedProcess.stdout);
+    sharedProcessLogger.pipeProcessOutputIntoLogger(this.sharedProcess.stderr);
```

Comment:
Unfortunately, doing this would break `resolveNetworkAddress` (it uses pipes to find grpc network address), so I would like to leave `stdio` as it is.

What comes to my mind is that shared process can work similarly to tshd, so it won't save any files on its own, but rather log everything to stdout. 

Cons:
- Stacktraces have time added to each line. 

Pros:
- We don't have to fight with duplicated logs.
- We have more control over the logs. As we have more logs sources, we can show source of each line in the terminal (it looks cool IMO):
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/20583051/199481847-9e3d1a05-123f-4636-a42d-da46cb729211.png">

Let me know guys what do you think!

Reply:
> it looks cool IMO

absolutely agree.

In terms of the con, I don't think adding time to each stack trace is too big of a deal. Sure, it adds some noise but as long as it doesn't _remove_ information it's not the worst. I like the idea of not saving any files on its own tbh.

Reply:
> What comes to my mind is that shared process can work similarly to tshd, so it won't save any files on its own, but rather log everything to stdout.

That seems to be a good solution.

You mentioned that time is added to each line but that doesn't seem to be the case on your screenshot, did you purposefully turn them off there or just cut them out when making the screenshot?